src/tests/multihost/alltests/test_krb_access_provider.py

```diff
@@ -0,0 +1,177 @@
+"""Automation for krb access provider
+"""
+from __future__ import print_function
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.exceptions import SSSDException
+from constants import ds_instance_name
+
+
+@pytest.mark.krbaccessprovider
+@pytest.mark.usefixtures('create_custom_posix_usersgroups')
+class TestKrbAccessProvider(object):
+    """
+    This is test case class for krb_access_provider suite
+    """
+    def test_0001_k5login_empty(multihost):
```

Comment:
Add @staticmethod decorator for methods so You do not have use self.
https://www.geeksforgeeks.org/python-staticmethod/